lib/restful_model_collection.rb

```diff
@@ -27,6 +27,11 @@ def each
       end
     end
 
+    def count
+      resp = RestClient.get(url, params: @filters.merge(view: 'count'))
+      JSON.parse(resp)['count']
```

Comment:
Could you refactor this code to be a little more like the rest of the SDK?
Notably, wrapping the response handling in a block and passing it to `Inbox.interpret_response` to handle errors.

See https://github.com/inboxapp/inbox-ruby/blob/master/lib/message.rb#L35 for an example